Help/Getting_In_Touch.rst

```diff
@@ -24,3 +24,10 @@ Modularity, clean good code as well as a high usability for both users
 and developers of analyse routines (called bears) stand in the
 foreground of the development. We will not speed up our development if
 it needs sacrificing any aspect of quality.
+
+
+Give Us Feedback
```

Comment:
Add the point below here: https://github.com/coala/documentation/pull/561/files#diff-15cbcde0631d4f2a91a45e04d0318b85R16

Reply:
don't need a new section

Reply:
but @anctartica can you see what @jayvdb had reviewed
he said me to add a new section

Reply:
he never mentioned a new section. 
> This should be after "- Give us feedback", as a new item

Reply:
where did he say to add a new section? @chhabrabhishek

Reply:
So what do I should change now?

Reply:
add a new point below the  "- Give us feedback" point

Reply:
and do i have to write report at community AT coala DOT io 
or do I simply write report at gitter and mail

Reply:
> If you believe someone is violating the code of conduct, we ask that you report it by emailing community AT coala DOT io.

Reply:
this is mentioned in COC

Reply:
Is it correct now? Or does it need any changes?